src/java/org/apache/cassandra/exceptions/RequestFailureReason.java

```diff
@@ -101,7 +100,15 @@ public static RequestFailureReason fromCode(int code)
 
     public static RequestFailureReason forException(Throwable t)
     {
-        return exceptionToReasonMap.getOrDefault(t.getClass(), UNKNOWN);
+        RequestFailureReason r = exceptionToReasonMap.get(t.getClass());
```

Comment:
@jacek-lewandowski , I made this without streams because in Apache Cassandra streams were forbidden in prod code recently.  I am trying to keep the code aligned as much as we can.